### ISSUE TITLE
Accessibility - Fix basic outline styles

### ISSUE THREAD

**Issue by slafleche** [vanilla/vanilla#6481]
The base styles for Vanilla unsets the default browser outline on elements. This causes problems for accessibility. We're also missing some `:focus` styles on links. We have some for `:hover`, we can duplicate those.

The change its self is quick and easy but requires a lot of testing. I'll need to make a spot check on all our core themes to make sure I'm not breaking anything. It's pretty much impossible to test every single theme, but it's not that drastic of a change change.

**Comment by slafleche** [comment]
Will be fixed with new theme(s)
